core/src/main/java/org/apache/carbondata/core/scan/result/vector/impl/directread/ColumnarVectorWrapperDirectWithDeleteDelta.java

```diff
@@ -58,6 +58,17 @@ public void putBoolean(int rowId, boolean value) {
     }
   }
 
+  @Override
```

Comment:
please check and handle for `ColumnarVectorWrapperDirectWithInvertedIndex`, `ColumnarVectorWrapperDirectWithDeleteDeltaAndInvertedIndex` also

Reply:
Done. Issue is with regards to only update query so changes may not be required in ColumnarVectorWrapperDirectWithInvertedIndex.